manifests/m/muCommander/muCommander/0.9.7/muCommander.muCommander.installer.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: muCommander.muCommander
+PackageVersion: 0.9.7
+Installers:
+- Architecture: x86
+  InstallerType: nullsoft
+  InstallerUrl: https://github.com/mucommander/mucommander/releases/download/0.9.7-1/mucommander-0.9.7-1-bundled.exe
+  InstallerSha256: DEC3929AA377122B08AE6205970A36F63CB24C065E4616901FC5A55056AE1E2C

```

Since the Publisher is missing we should add the ProductCode to make it match
```suggestion
  InstallerSha256: DEC3929AA377122B08AE6205970A36F63CB24C065E4616901FC5A55056AE1E2C
  ProductCode: 'muCommander'
```